README.md

```diff
@@ -51,6 +78,33 @@ Quick Start
 Trouble with these steps?  Read `TROUBLESHOOTING.md`.  More detailed build
 guidance is available below.
 
+**Submodule**
+
+If your project build tool is CMake, you can add libgit2 as a dependency on your project using `Git submodule`
```

Comment:
Proposal:

"If your project is using CMake as its build tool, you can add libgit2 as a dependency by embedding it as a Git 
submodule and then include it as a subdirectory in your build instructions."

Reply:
Furthermore, a ":" is missing at the end